test/unit_tests/util/messageRendererSpec.js

```diff
@@ -190,8 +190,34 @@ describe('renderMessage', () => {
     );
   });
 
+  it('renders a link from markdown notation with formatting', () => {
+    expect(renderMessage('[**doop**](http://www.example.com)')).toBe(
+      '<a href="http://www.example.com" target="_blank" rel="nofollow noopener noreferrer" data-md-link="true" data-uie-name="markdown-link"><strong>doop</strong></a>',
+    );
+  });
+
+  it('renders a without protocol link from markdown notation', () => {
+    expect(renderMessage('[doop](www.example.com)')).toBe(
+      '<a href="https://www.example.com" target="_blank" rel="nofollow noopener noreferrer" data-md-link="true" data-uie-name="markdown-link">doop</a>',
+    );
+  });
+
   it('does not render a js link from markdown notation', () => {
     expect(renderMessage("[doop](javascript:alert('nope'))")).toBe("[doop](javascript:alert('nope'))");
+    expect(renderMessage("[doop](javaScript:alert('nope'))")).toBe("[doop](javaScript:alert('nope'))");
```

Comment:
Not sure I see the diff here 🤔

Reply:
The uppercase `S` in the second test case